mobile/status.go

```diff
@@ -2261,3 +2261,13 @@ func addCentralizedMetric(requestJSON string) string {
 
 	return metric.ID
 }
+
+func Panic(message string) string {
```

Comment:
```suggestion
func IntendedPanic(message string) string {
```

Reply:
Had the same in mind, but was unsure.
Let's make it `IntendedPanic` indeed then.